coursier.bzl

```diff
@@ -82,6 +82,11 @@ sh_binary(
 )
 """
 
+def json_parse(json_string, fail_on_invalid = True):
+    if getattr(native, "json_decode", None) and not fail_on_invalid:
```

Comment:
Could you comment with the minimum bazel version json_decode is available in?

Reply:
Done.